### PR TITLE
Rx/staging

### PR DESCRIPTION
**Pull request by pyr0gan**
- Reviewed The Graph pathway steps 1, 2, 3

### CODE REVIEW

File: figment-learn/new-pathways/the_graph/the-graph-01-run-a-graph-node.md [hunk 1]
```diff
@@ -21,26 +25,42 @@ cd docker
 sudo ETHEREUM_RPC=mainnet:https://eth-mainnet.alchemyapi.io/v2/<API-KEY> docker-compose up
```
Comment (zurgl) [inline figment-learn/new-pathways/the_graph/the-graph-01-run-a-graph-node.md:25]: We should remove the `sudo` and replace docker-compose up by `docker compose up`

**Review by zurgl — CHANGES_REQUESTED** [review]
@pyr0gan Thanks, for your review. Just a minor change to do and it's ok for me.

File: figment-learn/new-pathways/the_graph/the-graph-01-run-a-graph-node.md (hunk 1 above)
Comment (pyr0gan) [reply figment-learn/new-pathways/the_graph/the-graph-01-run-a-graph-node.md]: 👍 done!

File: figment-learn/new-pathways/the_graph/the-graph-setup.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ To get your alchemy API key you:
 - Create a new App on Alchemy.
 - From the Dashboard, click on **View Details** and then click on **View Key**. You can then click to copy your new Alchemy API, including the endpoint URL. Paste this API key in a safe place.
 
-![](../../../.gitbook/assets/pathways/the-graph/graph-alchemy-setup.gif)
```
Comment (zurgl) [inline figment-learn/new-pathways/the_graph/the-graph-setup.md]: not sure it's a good idea

**Pull request merged**